test/SPI/private_swiftinterface.swift

```diff
@@ -34,14 +34,6 @@
 // RUN: %target-swift-frontend -typecheck-module-from-interface -I %t %t/Main.swiftinterface
 // RUN: %target-swift-frontend -typecheck-module-from-interface -I %t %t/Main.private.swiftinterface -module-name Main
 
-/// Serialize and deserialize this module, then print.
```

Comment:
I removed this because it interfered with the stability of printing the interface with an enum case declaration like `case a, b`. It seemed safe to remove to me because there are many other tests exercising serialization and deserialization of the `@_spi` attribute simply by compiling a helper module with `@_spi` declarations. If this is still needed for some reason, though, I can split it into a new test case.